app/views/articles/_actions.html.erb

```diff
@@ -40,9 +40,9 @@
             class="flex justify-between crayons-link crayons-link--block w-100 bg-transparent border-0"
             data-postUrl="<%= article_url(@article) %>">
             <span class="fw-bold"><%= t("core.copy_link") %></span>
-            <%= inline_svg_tag("copy.svg", aria_hidden: true, id: "article-copy-icon", class: "crayons-icon mx-2 shrink-0", title: "Copy article link to the clipboard") %>
+            <%= inline_svg_tag("copy.svg", aria_hidden: true, id: "article-copy-icon", class: "crayons-icon mx-2 shrink-0", title: t("core.copy_article")) %>
```

Comment:
I think a more re-usable "Copy to clipboard" would be a good adjustment here. Trying to limit sprawl and I could see it being useful to generalize this title as it could be used elsewhere.

So why Don't we go `core.copy_to_clipboard` and "Copy to clipboard" as the text here? I think contectually that would be okay.

Definitely open to more thoughts here.

Reply:
I guess the title there is appropriate as it's specifically saying to copy an article link. If we change it to copy to clipboard it will be common.
If the change is required I'll push a commit.